.github/workflows/build.yml

```diff
@@ -8,7 +8,7 @@ on:
 
```

Comment:
The commit message and PR description are missing a link to the hereby addressed GH Issue #.